integration/distinct_test.go

```diff
@@ -32,29 +32,29 @@ func TestDistinctErrors(t *testing.T) {
 	ctx, coll := setup.Setup(t, shareddata.Scalars, shareddata.Composites)
 
 	for name, tc := range map[string]struct {
-		command  any                // required
-		collName any                // optional
-		filter   any                // required
-		err      mongo.CommandError // required
+		command  any // required, command to run
+		collName any // optional, defaults to coll.Name()
+		filter   any // required
+
+		err        *mongo.CommandError // optional
+		altMessage string              // optional, alternative error message
+		skip       string              // optional, skip test with a specified reason
 	}{
-		"EmptyFilter": {
-			command: "a",
-			filter:  nil,
-		},
 		"StringFilter": {
 			command: "a",
 			filter:  "a",
-			err: mongo.CommandError{
+			err: &mongo.CommandError{
 				Code:    14,
 				Name:    "TypeMismatch",
 				Message: "BSON field 'distinct.query' is the wrong type 'string', expected type 'object'",
 			},
+			altMessage: "BSON field 'distinct.query' is the wrong type 'string', expected type 'object'",
```

Comment:
This `altMessage` is identical to the error message.
```suggestion
```

Reply:
Also for the same reason https://github.com/FerretDB/FerretDB/pull/2806/files#r1222350133